### PR TITLE
Meetings helm repo

### LINKED ISSUE #12: Helm charts for meetings and office
to be included in the upcoming version of the marketplace over VDCs

it's required to have
- [x] meetings chart
- [x] office chart

related`;

- [ ] icon for filebrowser https://github.com/threefoldtech/vdc-solutions-charts/issues/32

### PR DESCRIPTION
**Pull request by tobiaschielens**

### CODE REVIEW

**Review by m-motawea — DISMISSED** [review]
please add `readme` to explain the required config for the chart

you can use https://github.com/threefoldtech/vdc-solutions-charts/blob/master/charts/kube-prometheus-stack/README.md as an example

**Review by AhmedHanafy725 — DISMISSED** [review]
can you please follow these [steps](https://github.com/threefoldtech/vdc-solutions-charts#addingmodifying-a-chart-in-the-repo) for adding the chart?

**Comment by tobiaschielens** [comment]
Added readme + added resource limits

**Comment by tobiaschielens** [comment]
Added the steps  from https://github.com/threefoldtech/vdc-solutions-charts#addingmodifying-a-chart-in-the-repo

**Comment by ken-de-moor** [comment]
To be reviewed

File: charts/meetings/values.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: NodePort
```
Comment (xmonader) [inline charts/meetings/values.yaml]: NodePort service on 80? by default nodeports are on the range of 30k range How did you achieve that?

Comment (tobiaschielens) [reply charts/meetings/values.yaml]: As the old marketplace used to work, every solution got an FQDN. There was a reverse proxy to achieve that. In the mindset that everything will still have its own FQDN, I just put a reverse proxy in front myself to make the port forward to 80 locally.

Comment (xmonader) [reply charts/meetings/values.yaml]: this is not the right way to do in kubernetes. in kubernetes your run your service in the cluster, and if you want to expose you have multiple ways
1- NodePort the easiest, but then it opens a port -random port- on a specific range based on the cluster config usually above 31000 on all nodes e.g
2- Using ingress which is kinda like your reverse proxy running on 80, 443 and that proxies the traffic to another service on the cluster (which is almost all cases)

But definitely, you won't run a reverse proxy machine somewhere and forward the traffic to a nodePort

**Comment by tobiaschielens** [comment]
The package works on the threefold vdc
helm install meetings ./ --set ingress.hosts[0]=meetings.jimber.io --set ingress.certresolver=gridca

**Comment by RafyAmgadBenjamin** [comment]
we need to package and regenerate the index for this package according to the docs to be able to merge this to `master` branch
https://github.com/threefoldtech/vdc-solutions-charts

**Review by RafyAmgadBenjamin — APPROVED** [review]

**Review by ashraffouda — APPROVED** [review]

**Pull request merged**